parser/charset/encoding_gbk.go

```diff
@@ -55,6 +55,20 @@ func (e *encodingGBK) Peek(src []byte) []byte {
 	return src
 }
 
+func (e *encodingGBK) MbLen(bs string) int {
+	if len(bs) < 2 {
+		return 0
+	}
+
+	if 0x81 <= bs[0] && bs[0] <= 0xf4 {
```

Comment:
Could you add a comment/link about 2 bytes GBK characters?

Reply:
the 2 bytes character can refer to https://github.com/mysql/mysql-server/blob/3290a66c89eb1625a7058e0ef732432b6952b435/strings/ctype-gbk.cc#L3605